web/src/bootstrap_typeahead.ts

```diff
@@ -302,7 +302,8 @@ class Typeahead<ItemType extends string | object> {
         }
     }
 
-    defaultUpdater(item: ItemType): ItemType {
+    defaultUpdater(item: ItemType): string {
+        assert(typeof item === "string");
         return item;
     }
```

Comment:
More importantly, it looks like the original type was wrong. @evykassirer FYI, this tweak looks correct but please verify.